src/main/kotlin/interactor/GetCitiesHasTheCheapestBananaPricesInteractor.kt

```diff
@@ -0,0 +1,22 @@
+package interactor
+
+import model.CityEntity
+
+
+class GetCitiesHasTheCheapestBananaPricesInteractor ( private val dataSource: CostOfLivingDataSource ){
+
+
+    fun execute(vararg cities: CityEntity): List<String> {
+
+        return dataSource
+                .getAllCitiesData()
+                .filter (:: excludeNullAndNegativeBananaPrices)
+                .sortedBy { it.fruitAndVegetablesPrices.banana1kg }
+                .map { it.cityName }
+                .ifEmpty { throw Exception ("The List is empty.") }
```

Comment:
i think here its okay to return an empty list rather than throwing an exception.
so just delete line 16.

Reply:
ok